nextcord/message.py

```diff
@@ -543,6 +545,56 @@ def flatten_handlers(cls):
     return cls
 
 
+class MessageInteraction:
+    """Represents a message's interaction data, regardless of application.
```

Comment:
I feel it's a bit redundant given the extended description
```suggestion
    """Represents a message's interaction data.
```